### DISCUSSION TITLE
Install metrics

### DISCUSSION THREAD

**Discussion by zregvart** [redhat-openshift-ecosystem/community-operators-prod#667] [Q&A] [answered]
Hi :wave:
Are there any metrics data available for the community operators? For the Syndesis community, it would be interesting to see how many installs are there per version.

**Comment by J0zi** [comment, marked as answer]
Hi, we have information about bundle pulls, currently 2 daily. See https://quay.io/repository/openshift-community-operators/syndesis. Hover over blues squares on `Repository activity`.

**Reply by jahnavi2k** [reply]
Hi @J0zi 
Thanks for the information. Is there also a way to see number of installs from the OpenShift console rather than just the number of bundle pulls?

Thanks!
